cloudformation.tf

```diff
@@ -15,26 +8,24 @@ resource "aws_cloudformation_stack" "sns" {
 }
 
 resource "aws_cloudformation_stack" "datapipeline" {
-  count         = "${length(var.efs_ids)}"
-  name          = "efs-backup-datapipeline-stack-${count.index}"
+  count         = "${length(keys(var.efs_id_mpoint))}"
+  name          = "${module.tf_label.id}-${count.index}"
   template_body = "${file("${path.module}/templates/datapipeline.yml")}"
 
   parameters {
-    instance_type              = "${var.datapipeline_config["instance_type"]}"
-    subnet_id                  = "${data.aws_subnet_ids.all.ids[0]}"
-    security_group_id          = "${aws_security_group.datapipeline.id}"
-    efs_id                     = "${data.aws_efs_file_system.by_id.*.id[count.index]}"
-    efs_source                 = "${data.aws_efs_file_system.by_id.*.id[count.index]}.efs.${var.region}.amazonaws.com}"
-    #efs_source                 = "${aws_efs_mount_target.efs_target.*.dns_name[count.index]}"
-    s3_backups_bucket          = "${aws_s3_bucket.efs_backups.bucket_domain_name}"
-    time_zone                  = "${var.datapipeline_config["timezone"]}"
-    image_id                   = "${data.aws_ami.amazon_linux.id}"
-    topic_arn                  = "${aws_cloudformation_stack.sns.outputs["TopicArn"]}"
-    s3_log_bucket              = "${aws_s3_bucket.s3.id}"
-    datapipeline_resource_role = "${aws_iam_instance_profile.datapipeline_resource.name}"
-    datapipeline_role          = "${aws_iam_role.datapipeline_role.name}"
-    key_pair                   = "${var.ssh_key_name}"
-    period                     = "${var.datapipeline_config["period"]}"
-    tag                        = "efs-backup-${count.index}"
+    myInstanceType             = "${var.datapipeline_config["instance_type"]}"
```

Comment:
No, because `Parameter id must have prefix 'my'`
```
Error applying plan:

2 error(s) occurred:
* aws_cloudformation_stack.datapipeline[1]: 1 error(s) occurred:
* aws_cloudformation_stack.datapipeline.1: 
UPDATE_ROLLBACK_COMPLETE: ["Pipeline Definition failed to validate because of following Errors: [{ObjectId = 'tfInstanceType', errors = [Parameter id must have prefix 'my'
```